src/compositions/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,30 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Type } from "@compositions/FieldsetHeader/FieldsetHeader";
+import { FormControl } from "@compositions/FormControl/FormControl";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
+import { TextInput } from "@elements/TextInput/TextInput";
+import { action } from "@storybook/addon-actions";
+import { Meta, Story } from "@storybook/react";
+import { Accordion as AccordionComponent, AccordionItem, AccordionProps } from "./Accordion";
+
+export default {
+    title: "Compositions/Accordion",
+    component: AccordionComponent,
+} as Meta<AccordionProps>;
+
+export const Accordion: Story<AccordionProps> = () => (
+    <AccordionComponent>
+        <AccordionItem header={{ children: "Foo", decorator: <IconIcons />, type: Type.Accordion }}>
```

Comment:
I'd give the items more meaningful headers like "Accordion Type with Decorator", "AddRemove Type with onClick action" etc.

Reply:
7b53b09